### PR TITLE
add btc usd price

### PR DESCRIPTION
**Pull request by jeremy-babylonlabs**

### CODE REVIEW

File: src/app/components/Stats/Stats.tsx [hunk 1]
```diff
@@ -135,7 +137,14 @@ export const Stats: React.FC = () => {
       {
         title: "Confirmed TVL",
         value: stakingStats?.activeTVLSat
-          ? `${maxDecimals(satoshiToBtc(stakingStats.activeTVLSat), 2)} ${coinName}`
+          ? `${maxDecimals(satoshiToBtc(stakingStats.activeTVLSat), 2)} ${coinName}${
+              stakingStats.btcPriceUsd
```
Comment (vitsalis) [inline src/app/components/Stats/Stats.tsx:141]: On testnet this should always be `$0`

Comment (vitsalis) [reply src/app/components/Stats/Stats.tsx:141]: I don't mind not showing at all. What's important is that we do not show any real dollar values next to test tokens.

File: src/utils/formatAmount.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+export function formatAmount(amount: number): string {
```
Comment (totraev) [inline src/utils/formatAmount.ts:1]: Use build in [formatters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat)

**Review by totraev — APPROVED** [review]

File: src/app/components/Stats/Stats.tsx (hunk 1 above)
Comment (jeremy-babylonlabs) [reply src/app/components/Stats/Stats.tsx:141]: understood, I will merge this PR as is for now. In the public facing testnet we will not add the `btcPriceUsd` in the config from the api side cc:@filippos47

**Pull request merged**